src/pages/messages/components/NviCandidatePage.tsx

```diff
@@ -52,6 +71,24 @@ export const NviCandidatePage = () => {
         <ErrorBoundary>
           <ErrorBoundary>
             <PublicRegistrationContent registration={registrationQuery.data} />
+
+            {nextCandidateIdentifier && offsetNextCandidate && (
+              <MuiLink
+                sx={{ justifySelf: 'end' }}
+                component={Link}
```

Comment:
(placeholder-kommentar for å fikse hover/anchor/link til knappen, som vi kan resolve etter løsning)